.travis.yml

```diff
@@ -2,8 +2,6 @@ language: node_js
 services: mongodb
 sudo: required
 node_js:
-  - "4"
-  - "5"
```

Comment:
Let's add this in a separate PR?

Reply:
This whole branch will be replacing the old one so there isn't much point in that.